docs/HowTo/Interact/APIs/Authentication.md

```diff
@@ -126,34 +134,59 @@ expires, you need to generate a new token.
 
 Enable authentication from the command line and supply the external JWT provider's public key.
 
-JWT public authentication disables the Besu `/login` endpoint, meaning
-[username and password authentication](#username-and-password-authentication) will not work.
+!!! important
+    JWT public authentication disables the Besu `/login` endpoint, meaning
+    [username and password authentication](#username-and-password-authentication) will not work.
 
 ### 1. Generate a private and public key pair
 
 !!!note
-
     This step is for demonstration or testing purposes only. In a production environment the
     external JWT provider supplies the public key file.
 
 The private and accompanying public key files must be in `.pem` format.
 
-The key must use an RSA private key of at least 2048 bits.
+The key can be:
 
-!!! example "Sample using OpenSSL"
+* an RSA private key of at least 2048 bits algorithm also named `RS256`. This is the Besu default algorithm.
+* an ECDSA private key, algorithm also named `secp256r1` or `ES256`.
```

Comment:
hmmm what about RS384 512  ECDSA also supports other curve signatures other than `secp256r1`, `secp256k1` is an example

Reply:
Oh, that, I don't know, you told me only about RS256... Should I add more supported values or only talk about a higher level algo family? What would you suggest here?

Reply:
So we support `RS256`, `RS384`, `RS512` for RSA, and `ES256`, `ES384` and `ES512` for ECDSA, the users will be responsible for swapping out the algorithm and generating the correct key pair we can just mention that we support them